src/main/java/com/google/sps/utilities/RealAvailabilityDatastore.java

```diff
@@ -32,6 +32,8 @@
 import com.google.appengine.api.datastore.Query.CompositeFilter;
```

Comment:
Should this stuff be handled by session tokens? I was going to start working on that soon.

Reply:
I thought this would still be a helpful check event after that is implemented just so that we can avoid injections. But maybe it can be deleted after sessions are implemented if they are able to handle this.